### DISCUSSION TITLE
Support for creating Dolphin service menus

### DISCUSSION THREAD

**Discussion by kpbaks** [nix-community/plasma-manager#347] [Q&A]
Is there support for creating [dolphin service menus](https://develop.kde.org/docs/apps/dolphin/service-menus/) i.e. contextual right-click menu actions?

**Comment by magnouvean** [comment]
No higher-level modules is supported for dolphin at the moment that I'm aware of. You maybe could try to edit configfiles manually using `programs.plasma.configFile` in the meantime though.

**Reply by kpbaks** [reply]
Perfect, I will use that
